tests/components/rfxtrx/test_init.py

```diff
@@ -118,3 +124,31 @@ async def test_ws_device_remove(hass, hass_ws_client):
 
     # Verify that the config entry has removed the device
     assert mock_entry.data["devices"] == {}
+
+
+async def test_connect(hass):
+    """Test that we attempt to connect to the device."""
+    entry_data = create_rfx_test_cfg(device="/dev/ttyUSBfake")
+    mock_entry = MockConfigEntry(domain="rfxtrx", unique_id=DOMAIN, data=entry_data)
+
+    with patch.object(rfxtrxmod, "Connect") as connect:
+        mock_entry.add_to_hass(hass)
```

Comment:
We can move this up out of the patch context manager.

Reply:
https://github.com/home-assistant/core/pull/67278/files#diff-8330b0a7abb34e55d74b728522a45c9ba5546111542c82d8b4b4a3b106355dc4L134

https://github.com/home-assistant/core/pull/67278/files#diff-8330b0a7abb34e55d74b728522a45c9ba5546111542c82d8b4b4a3b106355dc4L148